librz/include/rz_il/definitions/bitvector.h

```diff
@@ -28,20 +28,24 @@ typedef enum {
  *  Ref : https://web.cs.dal.ca/~jamie/UWO/BitVectors/README.html
  */
 typedef struct bitvector_t {
-	ut8 *bits; ///< bits data
-	ut32 _elem_len; ///< length of ut8 array -- real / physical
+	union {
+		ut8 *large_a; ///< array of bytes for bitvectors > 64 bits whose size is defined in _elem_len
+		ut64 small_u; ///< value of unsigned small for bitvectors <= 64 bits
+		st64 small_s; ///< value of signed small for bitvectors <= 64 bits
```

Comment:
`small_s` is not going to work as expected. The sign bit on C-level will always be bit 63, but for interpreting a bv <64bits, this will make no sense. Imo this can be dropped and `small_u` always used instead with appropriate conversions.

Reply:
Removed.